### ISSUE TITLE
Extractor 2.3.0 release in TER is missing Libraries folder (getid3 library)

### ISSUE THREAD

**Issue by chesio** [xperseguers/t3ext-extractor#67]
I investigated why I'm suddendly getting errors below on one of my websites and found out that version 2.3.0 of Extractor in TER has no _Libraries_ folder, so `james-heinrich\getid3` library is not available:

```
Message: Core: Exception handler (WEB: BE): Error, code #0, file /[...]/typo3conf/ext/extractor/Classes/Service/Php/PhpService.php, line 169: Class "getID3" not found
```

The _Tests_ folder is present instead - my guess is that the `extension-release` script from `composer.json` wasn't run before upload of 2.3.0 release to TER.

**Comment by xperseguers** [comment]
Please check this pre-release and report.
[extractor_2.3.1.zip](https://github.com/xperseguers/t3ext-extractor/files/13238739/extractor_2.3.1.zip)

**Comment by chesio** [comment]
Seems to be working fine - I've tested it with TYPO3 11.5.32 and MP3 file with some ID3 tags.

**Comment by xperseguers** [comment]
Fixed with commit cba0a1630812e8351e17f3ef32654e48a7e6acdd
